core/server/data/db/connection.js

```diff
@@ -26,6 +26,22 @@ function configure(dbConfig) {
         dbConfig.useNullAsDefault = dbConfig.useNullAsDefault || false;
     }
 
+    if (client === 'mysql') {
+        dbConfig.timezone = 'UTC';
+    }
+
+    // https://github.com/tgriesser/knex/issues/97
+    // this sets the timezone to UTC only for the connection!
+    if (client === 'pg') {
```

Comment:
`config.database.client === 'pg' || config.database.client === 'postgres' || config.database.client === 'postgresql'`

or write a helper fn to check `config.database.client.isPG`?